android/src/main/java/com/reactnativetwiliovideo/TwilioVideoModule.kt

```diff
@@ -236,6 +236,7 @@ class TwilioVideoModule(reactContext: ReactApplicationContext) : ReactContextBas
         }
       }
       localVideoTrack.destroyFromReact()
+      promise.resolve(true)
```

Comment:
🤦

Reply:
ohhh haha this is why it times out lol nice